src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -559,5 +562,38 @@ private static string GetConversion(CodeWriter writer, CSharpType from, CSharpTy
             var propertyName = property.PropertyName;
             return writer.Scope($"if ({propertyName} != null)");
         }
+
+        public static IDisposable WriteCommonMethodWithoutValidation(this CodeWriter writer, MethodSignature signature, FormattableString? returnDescription, bool isAsync, bool isPublicType, bool enableAttributes = false, IEnumerable<Attribute>? attributes = default)
```

Comment:
I believe we do not need this `enableAttributes` parameter. We could either use null value or empty value of the array `attributes` to state that we do not want to write attributes

Reply:
This is actually shared with mgmt plane that does have references to it.  If we want to refactor that later we can but trying to keep the changes to other libraries at a minimum in this pr